src/Components/SelfInterview/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { Question } from "../../Apis/types";
+
+export const sortMethod = (questions: Question[], switchNumber: number) => {
+    let result = questions.sort((a: any, b: any) => {
+        if (a.question < b.question) return -1;
+        if (a.question > b.question) return 1;
+        return 0;
+    });
+
+    for (let i = 0; i < result.length; i += switchNumber) {
+        let a = result[i];
+        let b = result[i + 1];
+
+        result[i] = b;
+        result[i + 1] = a;
+    }
```

Comment:
This switches every switchNumber with the next item including 0. The instructions called for every third item to be put at the end of the array. Ex - 

[1,2,3,4,5,6,7,8] => [1,2,4,5,7,8,5,3]

Yours on the other hand will produce

[2,1,3,5,4,6,8,7]